gulpfile.js/tasks/build-sass.js

```diff
@@ -18,6 +18,10 @@ const buildSass = () =>
     .pipe(rename({ basename: 'standalone' }))
     .pipe(dest(outDir))
     .pipe(postcss([cssnano()]))
+    // Cssnano minimizes alpha colors to 8-digit hex codes, which is still stage 2 and is not supported in some browsers
+    // There didn't seem to be an easy way to disable this minification within cssnano,
+    // and it doesn't seem to work to add the postcss-color-hex-alpha plugin in the same postcss pass as cssnano
+    .pipe(postcss([require('postcss-color-hex-alpha')()]))
```

Comment:
> Could we disable the colormin option in cssnano? [cssnano.co/docs/optimisations/colormin](https://cssnano.co/docs/optimisations/colormin/)
I think I tried that and it didn't seem to be working.

> Or maybe we polyfill this down the line instead?

Do you mean transpile it in the WP site? Not completely sure what you mean.

Reply:
> Do you mean transpile it in the WP site? Not completely sure what you mean.

Yep! Kinda like what we were discussing for logical property support.

Reply:
(Also open to cssnano alternatives if it isn't working for us!)

Reply:
> I tried that and it didn't seem to be working.

Oops I just looked again at the configuration and realized that I was passing `colormin: false` directly to cssnano instead of to the default preset in cssnano. I can change it to do that and it works.

> Yep! Kinda like what we were discussing for logical property support.
Like I said in slack, I prefer to publish the latest fully standardized syntax, which rgba hex is still only stage 2. Since it is easy to transpile it without loss of functionality, I think it makes sense to do that transpilation here instead of in consuming repos.